lib/operations/execute_operation.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-const MongoError = require('../core').MongoError;
+const MongoError = require('../core/error').MongoError;
```

Comment:
why the diff here?

Reply:
there was a circular reference I think due to `ReadPreference`. Better to directly include files rather than use the single export from core now I think. That was just a stopgap to make import easier